src/refactor_nrepl/util.clj

```diff
@@ -41,3 +42,7 @@
     (if (seq xs)
       (apply conj coll xs)
       coll)))
+
+(defn normalize-var-name
+  [symbol-name-str]
```

Comment:
Just call the parameter `sym` and then call `str` on it imo.  

This function belongs in `core`, I think.  `util` is intended for stuff which might be useful in any clojure project.